src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathMaxLatencySpec.groovy

```diff
@@ -96,6 +103,136 @@ class ProtectedPathMaxLatencySpec extends HealthCheckSpecification {
         9               | "both paths do not satisfy SLA"
     }
 
+    @Tidy
+    def "Able to create DEGRADED protected flow with max_latency strategy if maxLatency < protectedPathLatency < maxLatencyTier2"() {
+        given: "2 non-overlapping paths with 10 and 15 latency"
+        setLatencyForPaths(10, 15)
+
+        when: "Create a flow with protected path, maxLatency 11 and maxLatencyTier2 16"
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            allocateProtectedPath = true
+            maxLatency = 11
+            maxLatencyTier2 = 16  // maxLatency < pathLatency < maxLatencyTier2
+            pathComputationStrategy = PathComputationStrategy.MAX_LATENCY.toString()
+        }
+        northboundV2.addFlow(flow)
+
+        then: "Flow is created, main path is the 10 latency path, protected is 16 latency"
+        and: "Flow goes to DEGRADED state"
+        wait(WAIT_OFFSET) {
+            def flowInfo =  northboundV2.getFlow(flow.flowId)
+            assert flowInfo.status == FlowState.DEGRADED.toString()
+            assert flowInfo.statusDetails.mainPath == "Up"
+            assert flowInfo.statusDetails.protectedPath == "degraded"
+            assert flowInfo.statusInfo == "A backup way was used to build the path"
+        }
+        def path = northbound.getFlowPath(flow.flowId)
+        pathHelper.convert(path) == mainPath
+        pathHelper.convert(path.protectedPath) == protectedPath
+
+        cleanup:
+        flow && flowHelperV2.deleteFlow(flow.flowId)
+    }
+
+    @Tidy
+    @Tags([LOW_PRIORITY])
+    def "Able to create DEGRADED flow with max_latency strategy if maxLatencyTier2 > pathLatency > maxLatency"() {
+        given: "2 non-overlapping paths with 11 and 15 latency"
+        setLatencyForPaths(11, 15)
+
+        when: "Create a flow with max_latency 11 and max_latency_tier2 16"
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            allocateProtectedPath = false
+            maxLatency = 11
+            maxLatencyTier2 = 16
+            pathComputationStrategy = PathComputationStrategy.MAX_LATENCY.toString()
+        }
+        northboundV2.addFlow(flow)
+
+        then: "Flow is created, main path is the 10 latency path, protected is 16 latency"
+        wait(WAIT_OFFSET) {
+            def flowInfo =  northboundV2.getFlow(flow.flowId)
+            assert flowInfo.status == FlowState.DEGRADED.toString()
+            assert flowInfo.statusInfo == "A backup way was used to build the path"
+        }
+        pathHelper.convert(northbound.getFlowPath(flow.flowId)) == protectedPath
+
+        cleanup:
+        flow && flowHelperV2.deleteFlow(flow.flowId)
+    }
+
+    @Tidy
+    @Tags([LOW_PRIORITY])
+    def "Able to update DEGRADED flow with max_latency strategy if maxLatencyTier2 > pathLatency > maxLatency"() {
```

Comment:
I'm not sure how maxLatencyTier2 applies to a non-protected flow. as far as I understand it is not used in any way

Reply:
PCE uses the `maxLatency` field, if the path was not found, then the `maxLatencyTier2` field is used.